src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Loader } from '../Loader';
+import { deleteTodo } from '../../api/todos';
+
+type Props = {
+  todo: Todo,
+  isTempLoading?: boolean,
+  setTodos?: (todos: Todo[]) => void,
+  pushError?: (title: string) => void,
+  todos?: Todo[],
+  toBeCleared?: Todo[],
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  isTempLoading,
+  setTodos = () => {},
+  pushError = () => {},
+  todos,
+  toBeCleared,
+}) => {
+  const { completed, title, id } = todo;
+  const [isBeingEdited] = useState(false);
```

Comment:
The state 'isBeingEdited' is initialized but never updated or used in the component. Consider removing it or implementing the editing functionality.